mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py

```diff
@@ -140,6 +140,9 @@ def test_email_subject_template_w_original(self):
         self.assertEquals(mail_message_1.subject, "Test and something more")
 
     def test_bad_template_does_not_break(self):
+        """
```

Comment:
Put only one line for this short docstring. Anyways, the method name is already enough, isn't it?